Configuration/PHPMD/rules.xml

```diff
@@ -42,7 +42,7 @@
     <!-- rules from the "naming" rule set -->
     <rule ref="rulesets/naming.xml/ShortVariable">
         <properties>
-            <property name="exceptions" value="id"/>
+            <property name="exceptions" value="id,lf"/>
```

Comment:
Why are only these two short var names allowed? Other valid short names may foreseeably be added, and limiting to two seems arbitrary.

Reply:
Generally, short variable names tend to be not very expressive (and we want our code to be expressive). So I'd like to have very few case-by-case exceptions to the "no very short names" rule.

I could move this from the configuration file to annotations in the code comments, though. What do you think, @samtuke?